src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas2/Cas2ApplicationTest.kt

```diff
@@ -309,22 +312,22 @@ class Cas2ApplicationTest : IntegrationTestBase() {
     }
 
     @Test
-    fun `Get list of applications returns 500 when a person cannot be found`() {
+    fun `Get list of applications returns successfully when a person cannot be found`() {
```

Comment:
You can set this test back to the way it was - apologies, I sent you down the wrong path with suggesting we don't error. We should still throw a 500...and I don't think we need anymore integration tests for this PR, as we haven't changed the end result of the code, just how we reach it

Reply:
actually ignore the above! I've realised as we are changing where we get the CRNs from we will have to change the stubbed API for integration tests 🙃